cves/kernel/CVE-2016-2184.yml

```diff
@@ -75,7 +79,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [971125]
```

Comment:
This didn't work when I tried to access it via [https://bugzilla.kernel.org/show_bug.cgi?id=971125](https://bugzilla.kernel.org/show_bug.cgi?id=971125). Was this perhaps on Red Hat's bugzilla?